arm/Microsoft.Synapse/workspaces/.parameters/encryptionwsai.parameters.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "value": "sxx-az-synws-encryptwsai-001"
+    },
+    "defaultDataLakeStorageAccountName": {
+      "value": "adpsxxazsaweux004"
+    },
+    "defaultDataLakeStorageFilesystem": {
+      "value": "synapsews"
+    },
+    "sqlAdministratorLogin": {
+      "value": "synwsadmin"
+    },
+    "encryption": {
+      "value": true
+    },
+    "encryptionKeyVaultName": {
+      "value": "carml-keyvault"
+    },
+    "encryptionKeyVaultResourceGroupName": {
+      "value": "platform-core-rg"
+    },
+    "encryptionKeyName": {
+      "value": "synapseEncryptionKey"
```

Comment:
Update to a key we have in the key vault - or create a new one via the dependency pipeline